main/getting-started/ertp-introduction.md

```diff
@@ -12,83 +12,96 @@ object. If it doesn't have a reference, it can't.
-quatloo brand mint.  Only mints can issue new digital assets. To mint
-new assets of a particular type, you must have a reference to that
-type's mint 
+In other words, a failed attempt to deposit a 3 Quatloo `payment`
+in a 7 Quatloo `purse` means the `payment` continues to exist and hold
+3 Quatloos, and the `purse` continues to hold 7 Quatloos.
 
-- **Issuers**: Create empty purses and payments and map minted
- assets to them when assets are added or removed. 
- Issuers verify and move digital assets.
-
-An issuer's special admin facet is a Mint, and that Mint and Issuer are
- associated with each other. With a reference to an issuer, you can
- check the validity of a payment in that issuer's mint's assets, as
- well as claim it either as a new payment to yourself or a purse you
- control. 
-
-Issuers (i.e. their references) should be gotten from a trusted source
-and then relied upon as the decider of whether an untrusted payment is
-valid  
+## Issuers and mints
 
-**Note**: There is a one-to-one correspondence between a brand and a
-  mint, a mint and an issuer, and an issuer and a brand. In other
-  words, each mint issues a unique kind of digital asset, say,
-  quatloos, and only that kind of asset.
+Other key ERTP components are:
+
+- **Mints**: Make new digital assets as a new `Payment`. `Mints` only
+make one kind of asset (Quatloos, Simoleons, Moola, other imaginary
+currencies, etc.). We refer to
+that kind as a `mint`'s *Brand*. So if a `mint` issues Quatloos, it's a
+Quatloo `brand` `mint`.  Only `mints` can issue new digital assets. To mint
+new assets of a particular kind, you must have a reference to that
+kind's `mint`. 
+
+- **Issuers**: Create empty `purses`, and manipulate and operate on
+ `payments`. `Issuers` are the authority on which `payments` and `purses`
+ hold what digital assets.
+
+An `issuer`'s special admin facet is a `mint`, and that `mint` and `issuer`
+have a one-to-one relationship. With a reference to an `issuer`, you can
+check the validity of a `payment` whose asset `brand` is associated with that `issuer`;
+i.e. If you have a reference to the Quatloos `issuer`, you can validate
+any `payment` made in Quatloos. You can also claim the `payment` either 
+as a new `payment` to yourself or a `purse` you control. 
+
+`Issuers` should be gotten from a trusted source
+and then relied upon as the decider of whether an untrusted `payment` is valid. 
+
+**Note**: There is a one-to-one correspondence between a `brand`, a
+  `mint`, and an `issuer`. In other words:
+  - A `mint` associated with a Quatloos `brand` can only create new Quatloos
+  and is the only `mint` that can create new Quatloos. 
+  - A `issuer` associated with a Quatloos `mint` can only operate on Quatloos
+  asset holders. It is the only `issuer` that can operate on them. 
 
 ## Amounts
 
 *Amounts* describe digital assets without having any value of their own.
 Anyone can make one, and they can be sent to anyone. They have two parts:
```

Comment:
```suggestion
*Amounts* describe digital assets without having any value of their own.
Anyone can make one, and they can be freely sent to anyone since they
convey no underlying value. They have two parts:
```